Parsers/ASimNetworkSession/Parsers/ASimNetworkSessionForcePointFirewall.yaml

```diff
@@ -0,0 +1,473 @@
+            "NSFNET-IGP","IGP",
+            "DCN-MEAS","DCN",
+            "STP","STP",
+            "SRP","SRP",
+            "HMP","HMP",
+            "XNET","XNET",
+            "VRRP","VRRP",
+            "ENCAP","ENCAP",
+            "CPNX","CPNX",
+            "PTP","PTP",
+            "SKIP","SKIP",
+            "SCPS","SCPS",
+            "Sprite-RPC","RPC",
+            "IPv6 ICMP","ICMP",
+            "MUX","MUX",
+            "CHAOS","CHAOS",
+            "SSCOPMCE","SSCOPMCE",
+            "CBT","CBT",
+            "SPS","SPS",
+            "ETHERIP","ETHERIP",
+            "MTP","MTP",
+            "ROHC","ROHC",
+            "CRTP","CRTP",
+            "PNNI","PNNI",
+            "NETBLT","NETBLT",
+            "TLSP","TLSP",
+            "IDPR","IDPR",
+            "DDX","DDX",
+            "PUP","PUP",
+            "DSR","DSR",
+            "NARP","NARP",
+            "CPHB","CPHB",
+            "SMP","SMP",
+            "L2TP","L2TP",
+            "IPv6 ICMP/143/0","ICMP",
+            "MICP","MICP",
+            "GMTP","GMTP",
+            "LARP","LARP",
+            "IFMP","IFMP",
+            "IGP","IGP",
+            "CFTP","CFTP",
+            "PGM","PGM",
+            "DDP","DDP",
+            "PIPE","PIPE",
+            "IATP","IATP",
+            "IGMP","IGMP",
+            "3PC","3PC",
+            "DGP","DGP",
+            "TCF","TCF",
+            "UTI","UTI",
+            "DCCP","DCCP",
+            "SWIPE","SWIPE",
+            "EMCON","EMCON",
+            "PIM","PIM",
+            "RVD","RVD",
+      ];
+    let ActionLookup=datatable(DeviceAction:string,DvcAction:string,EventResult:string,EventSeverity:string)
+    [
+              "Allow","Allow","Success","Informational",          
+              "Discard","Drop","Failure","Low",
+              "Permit","Allow","Success","Informational",          
+              "Refuse","Deny","Failure","Low",
+              "Terminate","Reset Source","Failure","Low",          
+              "Terminate (failed)","","Failure","Low",
+              "Terminate (passive)","Reset Destination","Failure","Low",          
+              "Terminate (reset)","Reset","Failure","Low",
+              "Wait for Authentication","","Success","Informational",
+              "Wait for Further Actions","","Success","Informational",          
+              "Wait for RPC Reply","","Success","Informational"
+    ];
+    let SubTypeLookup=datatable(DeviceEventClassID:string,EventSubType:string,DvcAction:string,EventResult:string,EventSeverity:string) //Add more codes if needed
+    [
+              "70018","Start","Allow","Success","Informational",          
+              "70019","End","Deny","Failure","Low", 
+              "70021","End","Reset","Failure","Low",          
+              "70022","End","Reset","Failure","Low"
+    ];
+    let EventResultMapping = datatable (Message:string, DvcAction:string, EventResult:string, EventResultDetails:string, EventOriginalResultDetails:string)  
```

Comment:
- I suspect many of those set EventSubType as well.
- Given that the Message is kept in the EventMessage field, I am not sure we need to set EventOriginalResultDetails